docs/sources/mimir/release-notes/v2.12.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Grafana Mimir version 2.12 release notes"
+menuTitle: "V2.12 release notes"
+description: "Release notes for Grafana Mimir version 2.12"
+weight: 1000
+---
+
+# Grafana Mimir version 2.12 release notes
+
+Grafana Labs is excited to announce version 2.12 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Added mimir-distroless docker image, built upon the distroless. This improvement is geared towards minimising attack surfaces and mitigating potential CVEs by trimming down the dependencies within the image. After comprehensive testing, we plan to shift from the default docker image to distroless.
+- **Added support to only count series that are considered _active_** through the Cardinality API endpoint `/api/v1/cardinality/label_names` by passing the `count_method` parameter. If set to `active` it counts only series that are considered _active_ according to the `-ingester.active-series-metrics-idle-timeout` flag setting rather than counting all in-memory series.
+- **Added new admin pages that display details about compaction jobs computed from bucket index**. They provide the same output as the `tools/compaction-planner` tool, without the need to provide all necessary configuration via CLI flags, since compactor already has all the required configuration. The new pages can be displayed by the following endpoints:
```

Comment:
I'm not sure this is note-worthy for release notes.

Reply:
I have removed this release-notes entry.